scripts/migration/schema-0.2-GPII-4014/verify.js

```diff
@@ -0,0 +1,165 @@
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script verifies updated CouchDB documents after the GPII-4014 deployment. It includes:
+// 1. Client credential documents have new fields added with proper values;
+// 2. All "schemaVersion" in documents have been set to 0.2. Note that "_design/views" doc doesn't have this field.
+// 3. All "timestampUpdated" have been set.
+
+// Usage: node scripts/migration/GPII-4014/verify.js CouchDB-url clientCredentialId-for-NOVA ...
+// @param {String} CouchDB-url - The url to the CouchDB where docoments should be verified.
+// @param {Strings} clientCredentialIds-for-NOVA - The "_id" value of the NOVA client credential. There could be any
+// number of client credential parameters from here onwards.
+
+// A sample command that runs this script in the universal root directory:
+// node scripts/migration/GPII-4014/verify.js http://localhost:25984 "clientCredential-nova1" "clientCredential-nova2"
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    url = require("url");
+
+fluid.registerNamespace("gpii.migration.GPII4014");
+
+require("./shared/migratedValues.js");
+require("../../shared/dbRequestUtils.js");
+require("../../../gpii/node_modules/gpii-db-operation/src/DbUtils.js");
+
+/**
+ * Create a set of options for this script.
+ * The options are based on the command line parameters and a set of database constants.
+ * @param {Array} processArgv - The command line arguments.
+ * @return {Object} - The options.
+ */
+gpii.migration.GPII4014.initOptions = function (processArgv) {
+    var options = {};
+    options.couchDbUrl = processArgv[2] + "/gpii";
+    options.novaClientCredentials = process.argv.splice(3);
+
+    // Set up database specific options
+    options.allDocsUrl = options.couchDbUrl + "/_all_docs?include_docs=true";
+    options.allDocs = [];
+    options.parsedCouchDbUrl = url.parse(options.couchDbUrl);
+    options.postOptions = {
+        hostname: options.parsedCouchDbUrl.hostname,
+        port: options.parsedCouchDbUrl.port,
+        path: "/gpii/_bulk_docs",
+        auth: options.parsedCouchDbUrl.auth,
+        method: "POST",
+        headers: {
+            "Accept": "application/json",
+            "Content-Length": 0,          // IMPORTANT: FILL IN PER REQUEST
+            "Content-Type": "application/json"
+        }
+    };
+    console.log("COUCHDB_URL: '" +
+        options.parsedCouchDbUrl.protocol + "//" +
+        options.parsedCouchDbUrl.hostname + ":" +
+        options.parsedCouchDbUrl.port +
+        options.parsedCouchDbUrl.pathname + "'"
+    );
+    return options;
+};
+
+/**
+ * Create the step that retrieves all documents from the database
+ * @param {Object} options - All docs URL and whether to filter:
+ * @param {Array} options.allDocsUrl - The url for retrieving all documents in the database.
+ * @return {Promise} - A promise that resolves retrieved documents.
```

Comment:
It doesn't resolve any documents, just to a message describing the verification results